features/extension/src/test/resources/wirings/9

```diff
@@ -0,0 +1,2 @@
+osgi.wiring.package; (&(osgi.wiring.package=org.osgi.framework)(version>=1.6.0)(!(version>=2.0.0)))
```

Comment:
This file doesn't contain the license header. You should exclude from rat configuration.